.travis.yml

```diff
@@ -1,34 +0,0 @@
-language: objective-c
-os: osx
-osx_image: xcode12
-before_install:
-  - echo -e "machine api.mapbox.com\n  login $SDK_REGISTRY_USER\n  password $SDK_REGISTRY_PASSWORD\n\n" > ~/.netrc
-  - echo -e "machine mapbox.bintray.com\n  login $BINTRAY_USER\n  password $BINTRAY_PASSWORD" >> ~/.netrc
-before_script: 
-  - echo $MAPBOX_API_KEY > .mapbox
-  - make dependencies
-script:
-  - xcodebuild -scheme Demo -destination platform\=iOS\ Simulator,name\=iPhone\ 11,OS\=latest clean test | xcpretty
-after_script:
-  - scripts/trigger_metrics.sh
-branches:
-  only:
-  - develop
-  - master
-  - /^release/.*$/
-
-before_deploy: xcodebuild -scheme "MapboxSearchUI" -destination platform=iOS\ Simulator,name=iPhone\ 11 clean test
```

Comment:
Don't we need these tests run on CI?

Reply:
It is a rudiment, we use CircleCI workflow to run all release checks.